### PR TITLE
Set continue_on_step_failure setting to False for FT, Eval, Import notebooks

### PR DESCRIPTION
**Pull request by skasturi**
# Description

Currently default setting for continue_on_step_failure is True resulting in further steps getting executed. Setting this to False for better customer experience.

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/python/foundation-models/system/import/import_model_into_registry.ipynb [hunk 1]
```diff
@@ -505,9 +508,9 @@
  "metadata": {
   "celltoolbar": "Tags",
   "kernelspec": {
-   "display_name": "Python 3.10 - SDK V2",
+   "display_name": "v2",
    "language": "python",
-   "name": "python310-sdkv2"
+   "name": "python3"
```
Comment (novaturient95) [inline sdk/python/foundation-models/system/import/import_model_into_registry.ipynb:513]: Is this change expected? Probably got updated with local terminal settings?

Comment (novaturient95) [reply sdk/python/foundation-models/system/import/import_model_into_registry.ipynb:513]: Should we also make sure to set kernel for finetuning and evaluation notebooks as well?

Comment (skasturi) [reply sdk/python/foundation-models/system/import/import_model_into_registry.ipynb:513]: Yes. Didn't make any explicit change. Thanks for catching. Will revert it.

Comment (skasturi) [reply sdk/python/foundation-models/system/import/import_model_into_registry.ipynb:513]: Done.

**Review by novaturient95 — APPROVED** [review]

**Review by rtanase — APPROVED** [review]

**Pull request merged**